server/models.py

```
@@ -1,15 +1,9 @@
-from sqlalchemy import Column, Integer, String, Text, ForeignKey, Table
+from sqlalchemy import Column, Integer, String, Text, ForeignKey, Table, ARRAY
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship, backref, Session
```

Ruff

`sqlalchemy.orm.Session` imported but unused

Remove unused import: `sqlalchemy.orm.Session`

(F401)